web_console/dynamic-plug-in/deploy-plug-in-cluster.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="deploy-plug-in-cluster_{context}"]
+= Deploy your plug-in on a cluster
```

Comment:
My ignorance: The other assemblies refer to _dynamic_ plug-ins. Here it is plug-in. I assume they are the same thing. Probably doesn't matter to a proper user. Just wanted to ask.

Reply:
yeah I just adjusted the tense based on the sentence